Gidraff/graphql-photo-share-api
# Configure development environment

#### What does this PR do?
Setup the development environment servers
#### Description of Task to be completed?
Have application and database server running
#### How should this be manually tested?
*simply follow the following steps:*
- Clone this repository.If all went well you should see `graphql-photo-share-api`.
- Change directory into the repo/directory
- Once inside run `npm install` to install dependencies.
- Before running `npm start` make sure you have mongo daemon is running(You need to install mongo if it doesn't already exist.
